source/content/terminus-2-0.md

```diff
@@ -6,20 +6,14 @@ categories: [platform]
 tags: [cli, drush, local, wp-cli, terminus]
 ---
 
-<Alert title="Note" type="info" >
+<Alert title="Warning" type="danger" >
```

Comment:
Actually, I'd love to remove this page and redirect to terminus-3-0 (https://pantheon.io/docs/terminus/terminus-3-0). Let me know if this is possible and how could I help to make it happen

Reply:
Agreed!